Assignment2/mcaminome-MacarenaCamino.ttl

```diff
@@ -0,0 +1,18 @@
+@base <http://www.oeg-upm.net/ontologies/class/ >
+@prefix class: <http://www.ontologies.org/ontologies/class#>
+@prefix computer: <http://www.ontologies.org/ontologies/computer#>
+@prefix user: <http://www.ontologies.org/ontologies/user#>
+@prefix sensor: <http://www.ontologies.org/ontologies/sensor#>
+@prefix measurement: <http://www.ontologies.org/ontologies/measurement#>
+
+:Class01	class:includes:Sensor029;
+			class:includes:Computer101.
+							
+	:Sensor029	sensor:hasMeasurement:Measurement8401.
+
+		:Measurement8401	measurement:hasTemperature:"29"^^xsd:integer;

```

Comment:
Have you tried with a validator? You should always try to leave a blank space (at least) between elements in the triple